ios/apptentive_flutter.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name             = 'apptentive_flutter'
-  s.version          = '5.8.3'
```

Comment:
So, 6.0.1 is apptentive flutter plugin version that uses iOS 6.0.3 & android 5.8.3?

Reply:
Correct